### ISSUE TITLE
Citation?

### ISSUE THREAD

**Issue by ViralBShah** [cscherrer/Soss.jl#19]
Is there a published paper or JOSS entry that can be cited? If so, would be great to have a CITATION.bib file in the package.

**Comment by cscherrer** [comment]
Good suggestion, thanks @ViralBShah . Nothing yet, but I'm hoping to get something together for JuliaCon 2019

**Comment by tamasgal** [comment]
Any updates on this?

**Comment by ViralBShah** [comment]
Even a JOSS submission for citation purposes would suffice.

**Comment by cscherrer** [comment]
Good idea, I'll make a JOSS submission top priority. @tamasgal do you have a submission deadline for a paper you're working on?

**Comment by tamasgal** [comment]
It'll not happen before March, so no hurry from my side 😉

**Comment by cscherrer** [comment]
We had been targeting Probprog (due Jan 10), but with a JOSS entry we could self-cite ;)

**Comment by cscherrer** [comment]
https://github.com/openjournals/joss-reviews/issues/1955
